babel.config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "plugins": [
+  ["@babel/plugin-transform-typescript", { "allExtensions": true, "onlyRemoveTypeImports": true, "allowDeclareFields": true }],
+    "@embroider/addon-dev/template-colocation-plugin",
+    ["babel-plugin-ember-template-compilation", {
+      "targetFormat": "hbs",
+      "transforms": []
+    }],
+    ["module:decorator-transforms", { "runtime": { "import": "decorator-transforms/runtime" } }],
```

Comment:
Should be removed, as decorator is not needed

Reply:
thanks! removed